build.gradle.kts

```diff
@@ -17,6 +18,17 @@ plugins {
     alias(libs.plugins.spotless) apply false
 }
 
+apiValidation {
+    @OptIn(kotlinx.validation.ExperimentalBCVApi::class)
+    klib {
+        enabled = true
+    }
+    ignoredProjects += setOf(
+        "test",
+        "coverage",
+    )
```

Comment:
No need to check test projects.

https://github.com/ComposeGears/Valkyrie/blob/a57c07eeb41799ef07e97961eb16307d0a13f3ba/settings.gradle.kts#L67